apps/emqx_oracle/src/emqx_oracle.erl

```diff
@@ -258,9 +258,7 @@ oracle_host_options() ->
     ?ORACLE_HOST_OPTIONS.
 
 connect(Opts) ->
-    Password = emqx_secret:unwrap(proplists:get_value(password, Opts)),
-    NewOpts = lists:keyreplace(password, 1, Opts, {password, Password}),
-    jamdb_oracle:start_link(NewOpts).
+    jamdb_oracle:start_link(Opts).
```

Comment:
Reminder: use `jambd_secret` to wrap rather than `emqx_secret`, for consistency.